src/shared/components/ContextMenu/ContextMenu.style.ts

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+import { colors, sizes, transitions, zIndex, typography } from '../../theme'
+import { Text, Icon } from '@/shared/components'
+
+type ContainerProps = {
+  isActive?: boolean
+  position: { x: number; y: number; left: boolean }
+}
+
+export const StyledContainer = styled.div<ContainerProps>`
+  position: absolute;
+  visibility: ${({ isActive }) => (isActive ? 'visible' : 'hidden')};
+  top: ${({ position }) => position.y}px;
+  left: ${({ position }) => position.left && position.x}px;
+  right: ${({ position }) => !position.left && position.x}px;
+  opacity: ${({ isActive }) => (isActive ? '1' : '0')};
+  transform: scale(${({ isActive }) => (isActive ? '1' : '0.88')});
+  background-color: ${colors.gray[800]};
+  width: 200px;
+  color: ${colors.white};
+  padding: ${sizes(2)};
+  z-index: ${zIndex.globalOverlay};
+  transition: opacity 200ms ${transitions.easing}, transform 200ms ${transitions.easing};
+`
```

Comment:
```suggestion
  word-break: break-all;
`
```
Also, there is one more minor issue(and I think it occurs also in Snackbar).
Look at the size of the Icon.
![image](https://user-images.githubusercontent.com/51168865/108051471-39d43a00-704b-11eb-8eb8-ed9dee08dba3.png)